Makes v3 the default, merge aligned with LLD/LLM prs

juan-cortes opened the pull request:
Both https://github.com/LedgerHQ/ledger-live-desktop/pull/3854 and https://github.com/LedgerHQ/ledger-live-mobile/pull/1745 need to depend on a live-common version including this switch before they can be merged.

juan-cortes committed:
Makes v3 the default, merge aligned with LLD/LLM prs

gre commented:
am i right to understand we want to merge this only when it's a GO? then i think we can tmp close the PR @juan-cortes

juan-cortes commented:
> am i right to understand we want to merge this only when it's a GO? then i think we can tmp close the PR @juan-cortes

Sure, close if it's clearer for you 🤷🏼

gre commented:
so when we have a go to wyre feature, we merge it on desktop/mobile with this? @juan-cortes

juan-cortes committed:
Fix double open Exchange app bug

We introduced a regression with the "requireLatestFirmware" logic where we would go over the flow without checking for dependencies, this commit addresses that issue.

juan-cortes commented:
> so when we have a go to wyre feature, we merge it on desktop/mobile with this? @juan-cortes

That is correct, we need this in a live-common version so that it's v3 prod by default. I also had to add a fix for a small bug that triggered the open app twice when we required the latest firmware, it didn't break the flow but it required an extra ok from the user, which is not great. Sorry about that
